browser/youtube_script_injector/android_youtube_script_injector_browsertest.cc

```diff
@@ -0,0 +1,101 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include <string>
+
+#include "base/path_service.h"
+#include "brave/browser/brave_content_browser_client.h"
```

Comment:
Look like we don't need this include.
Let's leave only what we actually use.

Reply:
Is there a check we can run to remove unused include?

Reply:
If you use clangd plugin it has ability to highlight unused includes.
Although, I don't know about any tool to check it automatically.